app/views/Notification.js

```diff
@@ -0,0 +1,292 @@
+      this.setState({ dataAvailable: false });
+    }
+  }
+
+  generate_random_intersections(length = 28) {
+    using_random_intersections = true;
+    var dayBin = [];
+    for (var i = 0; i < length; i++) {
+      // Random Integer between 0-99
+      const intersections = Math.floor(Math.random() * 100);
+      dayBin.push(intersections);
+    }
+    SetStoreData('CROSSED_PATHS', dayBin);
+    this.refreshState();
+  }
+
+  getInitialState = async () => {
+    GetStoreData('CROSSED_PATHS').then(dayBin => {
+      console.log(dayBin);
+      if (dayBin === null) {
+        this.setState({ dataAvailable: false });
+        console.log("Can't found Crossed Paths");
+      } else {
+        var crossed_path_data = [];
+        console.log('Found Crossed Paths');
+        this.setState({ dataAvailable: true });
+        dayBinParsed = JSON.parse(dayBin);
+        for (var i = 0; i < dayBinParsed.length; i++) {
+          const val = dayBinParsed[i];
+          data = { x: i, y: val, fill: this.colorfill(val) };
+          crossed_path_data.push(data);
+        }
+        this.setState({ data: crossed_path_data });
+      }
+    });
+  };
+
+  colorfill(data) {
+    var color = 'green';
+    if (data > 20) {
+      color = 'yellow';
+    }
+    if (data > 50) {
+      color = 'orange';
+    }
+    if (data > 80) {
+      color = 'red';
+    }
+    return color;
+  }
+
+  render() {
+    return (
+      <SafeAreaView style={styles.container}>
+        <View style={styles.headerContainer}>
+          <TouchableOpacity
+            style={styles.backArrowTouchable}
+            onPress={() => this.backToMain()}>
+            <Image style={styles.backArrow} source={backArrow} />
+          </TouchableOpacity>
+          <Text style={styles.headerTitle}>
+            {languages.t('label.private_kit')}
+          </Text>
+        </View>
+
+        <View style={styles.main}>
+          <Text style={styles.headerTitle}>
+            {languages.t('label.notification_title')}
+          </Text>
+          {this.state.dataAvailable ? (
+            <>
+              <VictoryChart height={0.35 * height} dependentAxis={true}>
+                <VictoryAxis dependentAxis={true} />
+
+                <VictoryBar
+                  alignment='start'
+                  style={{
+                    data: {
+                      fill: ({ datum }) => datum.fill,
+                    },
+                  }}
+                  data={this.state.data}
+                />
+              </VictoryChart>
+              <Text style={styles.mainText}>
+                {languages.t('label.notification_main_text')}
+              </Text>
+              <ScrollView contentContainerStyle={styles.contentContainer}>
+                {this.state.data.map(data => (
+                  <TouchableOpacity
+                    key={data.x}
+                    style={styles.buttonTouchable}
+                    onPress={() => this.alertItemName(data.y)}>
```

Comment:
This is undefined.  Please see attached.
<img width="392" alt="Screen Shot 2020-03-27 at 5 11 43 PM" src="https://user-images.githubusercontent.com/9043683/77804642-12007000-704e-11ea-8983-a1656759ec14.png">